py-core/app/util/database_functions.py

```diff
@@ -0,0 +1,118 @@
+import sqlite3
+
+
+DATABASE_PATH = "./node_database.db"
+
+
+def setup_database():
+    database = sqlite3.connect(DATABASE_PATH)
+    cursor = database.cursor()
+    command = """CREATE TABLE IF NOT EXISTS 'nodes' (
+        name VARCHAR PRIMARY KEY,
+        arronax_port INTEGER,
+        conseil_port INTEGER,
+        node_port INTEGER,
+        network VARCHAR,
+        history_mode VARCHAR,
+        status VARCHAR 
+        );"""
+    cursor.execute(command)
+    database.commit()
+
+
+def is_name_taken(name):
+    database = sqlite3.connect(DATABASE_PATH)
+    cursor = database.cursor()
+    command = """SELECT name FROM 'nodes' WHERE name="{}";""".format(name)
+    cursor.execute(command)
```

Comment:
Would be worth refactoring this repeated pattern of connecting, getting a cursor, and executing into its own func. [Dry](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code